CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.3.0
```

Comment:
Surround with [].

or i guess we could do that in release. upto you

Reply:
I will leave it for release 😀